extra/html-extra/Tests/CVATest.php

```diff
@@ -0,0 +1,661 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Html\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Twig\Extra\Html\Cva;
+
+class CVATest extends TestCase
```

Comment:
```suggestion
class CvaTest extends TestCase
```

Don't forget to rename the file as well.